cfgov/jinja2/v1/_includes/molecules/jobs-list.html

```diff
@@ -0,0 +1,57 @@
+{# ==========================================================================
+
+   jobs_list.render(value)
+
+   ==========================================================================
+
+   Description:
+
+   Creates jobs list when given:
+
+   value:              Object defined from a StreamField block.
+
+   value.heading:      The heading of the jobs list.
+                       Default is "Career Openings".
+
+   value.jobs:         A list of jobs with the link first
+                       followed by the text of link.
+
+   value.jobs[i].url:  A string for the URL of the link.
+
+   value.jobs[i].text: A string for the text of the link.
+
+
+   ========================================================================== #}
+
+{% macro render(value) %}
+<aside class="m-jobs-list"
+       data-qa-hook="openings-section">
+    <h3>
+        {{ value.heading or 'Career Openings' }}
+    </h3>
+    {% if value.jobs is defined and value.jobs | length > 0 %}
+    <ul class="list list__unstyled">
+        {% for job in value.jobs %}
+        <li class="list_item">
+            <a class="list_link"
+               href="{{ job.url }}">{{ job.text }}</a>
+            <p class="date">
+                CLOSING
+                {% import 'macros/time.html' as time %}
+                {{ time.render(job.close_date, {'date':true}) }}
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2275/commits/3d7f587c7d6662bf102212e0c712287298cb2fb6.